image-layout.md

```diff
@@ -0,0 +1,47 @@
+## Open Container Initiative Image Layout Specification
+
+The OCI Image Layout is a slash separated layout of OCI content-addressable objects and human significant references (refs) 
```

Comment:
“v1.0” is (without more context) not really significant to me.  I'd recommend “[location-addressable][1] references.” (with the trailing period too ;).

[1]: https://en.wikipedia.org/wiki/Content-addressable_storage#Content-addressed_vs._location-addressed